### PR TITLE
[Console] Inherit phpdoc from OutputFormatterInterface

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony/pull/22225#discussion_r109361374
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Console/Formatter/OutputFormatter.php [hunk 1]
```diff
@@ -79,56 +79,39 @@ public function __construct($decorated = false, array $styles = array())
...
     }
 
     /**
-     * Gets style options from style with specified name.
-     *
-     * @param string $name
-     *
-     * @return OutputFormatterStyleInterface
-     *
-     * @throws \InvalidArgumentException When style isn't defined
```
Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: let me know if this need to be added to the interface (see also #22231).

Comment (chalasr) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: I would say it should be. unrelated but this should uses the console domain exception instead

Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: personally i never add logic ones to the interface.. only concrete runtime scenarios that all implementations must adhere to (`SomethingIsWrongException`). But this is a thin line in PHP so i dont mind, hence i asked :)

Comment (chalasr) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: let's see what others think then :)

Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: to clarify.. i always follow; logic=programmers fault, runtime=program fault

Comment (ro0NL) [inline src/Symfony/Component/Console/Formatter/OutputFormatter.php]: given the actual implementation is a program fault.. i think im gonna add it. I would have used `StyleNotFoundException extends \RuntimeException`. But that's semantics.. who cares :P

**Review by nicolas-grekas — APPROVED** [review]
:+1:

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**